### DISCUSSION TITLE
Combining multiple kuztomize test results into single junit file

### DISCUSSION THREAD

**Discussion by noorul** [datreeio/datree#902] [Q&A] [answered]
We have multiple kustomize overlay folders. I found a helper script at https://hub.datree.io/integrations/kustomize-support but not sure how to get the JUnit results into a single XML file.

**Comment by eyarz** [comment, marked as answer]
The functionality that you describes is not supported.
Feel free to open an issue with a feature request or to submit a PR to add this functionality :)
